src/app/feed/feed.component.ts

```diff
@@ -537,7 +537,10 @@ export class FeedComponent implements OnInit, OnDestroy, AfterViewChecked {
       .pipe(
         tap(
           (res) => {
-            this.globalVars.hotFeedPosts = this.globalVars.hotFeedPosts.concat(res.HotFeedPage);
+            if (res.HotFeedPage) {
```

Comment:
fix a bug in dev test env. hot feed is returned as null and dumps unnecessary noise in the console.